test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignatureTrustAndValidityVerificationProviderTests.cs

```diff
@@ -676,7 +715,7 @@ public async Task GetTrustResultAsync_SettingsRequireExactlyOneTimestamp_Multipl
 
             using (var package = new PackageArchiveReader(await nupkg.CreateAsStreamAsync(), leaveStreamOpen: false))
             using (var testCertificate = new X509Certificate2(_trustedTestCert.Source.Cert))
-            using (var signatureRequest = new AuthorSignPackageRequest(testCertificate, HashAlgorithmName.SHA256))
+            using (var signatureRequest = new AuthorSignPackageRequest(testCertificate, NuGet.Common.HashAlgorithmName.SHA256))
```

Comment:
Consider using a name alias to avoid prefixing the type with the full name (namespace + type). In this case:

```suggestion
            using (var signatureRequest = new AuthorSignPackageRequest(testCertificate, HashAlgorithmName.SHA256))
```

For example, at the beginning of the file, you could add:

```c#
using HashAlgorithmName = NuGet.Common.HashAlgorithmName;
```

Reply:
Thanks! Fixed.